### PR TITLE
Makes vampires more containable

### PR DESCRIPTION
**Pull request by Allfd**
This PR provides for non-lethal vampire handling options, the purpose is to enable low-powered vampires to be able to be inserted into prison populations without posing a constant threat.

This will enable more IC options and RP possibilities for this class of antags as prisoners under the existing execution rules.

🆑Alffd
add: A new Anti-Bite muzzle has been added to the SecVendor. This muzzle can be locked through the strip panel and will prevent a vampire from feeding.
add: Holy water now causes vampires to vomit blood, before vomiting the water. After this holywater behaves as normal.
/🆑

### CODE REVIEW

**Comment by uraniummeltdown** [comment]
This is an unnecessary buff to holy water as blood is required to use any ability other than Glare and Rejuvenate. Also it only affects non full powered vamps who are low on blood anyway.
The muzzle is stupid as a full powered vamp will have killed people and will probably be shot instead of imprisoned.
The muzzle is just way too secure, easy and safe. A whole item that is locked to Security access for the sole purpose of stopping vamps from draining blood. I've seen vamps imprisoned before and they're hardly broken, so they might be able to glare and suck 30 or 40 blood from the officer sent to check up on them, so what?
It's an anti-fun item that will only affect low-power vamps who get caught and prevents any interesting ways of helping a vampire in prison like smuggling in monkey cubes from space. Locked to Sec access really? Even the perma jumpsuits can be taken off my anyone.

**Comment by TullyBurnalot** [comment]
@Allfd The completely unremovable mask is a tad on the excessive side for vampire containment. We'd prefer it if it were changed to allow other people to remove the muzzle, on a 1-2 minute timer.

**Comment by Doukhan** [comment]
No point to this as there is currently no such thing as antag/sec RP,  vampires are always KoS/dump in perma and ignore and sec are always murder whenever possible. These would only see use as griefing and torture items.

**Comment by imsxz** [comment]
Could make it like the electropack, where you can't remove it on your own, but others easily can. ID locked seems a bit much. At least this way, other prisoners could remove it to help the vampire escape.

**Comment by EldritchSigma** [comment]
Lets face it... if someone gets permaed they either SSD or commit suicide by hitting themself with anything aviable and using *succumb asap. Why is that? There is NO reason to stay alive locked in a room being able to do nothing whatsoever but growing plants with utterly inefficient tools. Sec barely interacts with you and only interact if you commit suicide and they want their valids. Speaking out of that one time I got permaed as a vamp thrall. Didnt work out for them because I got *succumb macroed.  Spectating or going SSD is literally better than sitting in perma doing nothing. And why granting them the satisfaction of keeping you alive when you can deny it to them with no consequences... Cause thats what almost everyone is doing.

**Comment by taukausanake** [comment]
It might not hurt to have people, maybe ghosts, whose job is to break people out of perma. Kind of like the bless/smite protector/assassin thing. The only major problem I can see is the possible damage or security holes left over from a break out. It would give people a reason to believe they aren't locked out from the round after perma

**Comment by Allfd** [comment]
@TullyBurnalot changes made with a 60 second timer.


To address the purpose behind this PR,  a number of other things are in the works to make Perma more interesting.  Suicide is always an option, but hopefully soon we will be at the point where that is no longer the only option.

**Comment by Dumbdumn5** [comment]
Not a fan of the muzzle or the water changes, really. The muzzle seems like something that, if combined with a straight jacket and enough holy water, will become pretty much the bane of any vamps existence. Vomiting up a significant portion of blood just from holy water consumption means zilch usable blood for containment, muzzle ensures you likely won't be getting any more usable blood, and security will likely slaughter you if you come anywhere near escaping.

Perma is difficult to make interesting, and making it even harder to escape from than it already was doesn't seem like a fix or positive change as much as it seems like this will just result in an easy way to totally depower even the most prepared of vamps and shove them in a box that they can totally forget them in. They can't break out into space, vamps burn there, not to mention oxy, brute, and burn. Nabbing a sec ID is impossible unless somebody dumbly walks into the perma/solitary cell they stuck you in. Getting enough blood to escape assumes you'll be around other prisoners, when it's probably not likely that you'll be placed in a room with another after being known as a vampire. On top of that, usage of holy water chem implants will probably make short work out of any successfully escaped vamp, rapidly draining their power and leaving they almost completely vulnerable aside from a glare, which won't effect the sunglass'd sec staff, and whatever they've found.

If perma were easier to escape, or if we had more ways to break out of perma, I'd understand the change. As it stands, my vote rests with :-1: .

**Comment by Allfd** [comment]
@Dumbdumn5 

Currently captured Vampires tend to just get thrown in solitary or killed on sight, the idea behind this is to give security enough confidence that a vampire is no-longer a threat to have that not happen.

Do you have something else you think would help with that?

**Comment by Dumbdumn5** [comment]
My largest issue is mostly with the holy water vomiting up usable blood, as that can take a lot of preparation (barring the current monkey BSing), and storing up blood for use in escape as a bit of a reserve should still be a valid tactic, as jaunt can only bring a player so far, and holy water prevents jaunting through a certain region if sprayed/dumped on the tile iirc (not tested recently, so I could be wrong and this may only apply to wraith phase shifting). The muzzle just seems a bit excessive, but it could be lived with. 

The problem with it is that security could feel way too confident, having removed pretty much any chance of the vampire escaping, and leaving them, if alone, as powerless as a civvie chucked into perma assuming they're smart enough to not walk in the unmuzzled vamp's cell alone.

Whatever the solution, the holy water change may be neutering them a bit too hard. Muzzle is debateable, but I'd be fine with having it about.

**Comment by Allfd** [comment]
@Dumbdumn5 Currently the minimum application of holywater, 5 (units) is a fatal dosage.  At the point the vampire is fed holy water they are essentially dead, this was changed by request to holy water I think partially for that reason.

**Comment by Dumbdumn5** [comment]
Being dead can be, in a way, better than being kept in perma by security with no way out and nothing to do. At least being a ghost gives you a chance for special roles. Perma needs to be a bit more interesting or easier to break out of/get broken out of by other crew to become something that should warrant more ways to get into it, especially something that totally neuters the ability of the vampire antag to even get out of the place. In terms of vampires in the field, loading up a syringe gun with holy water is a surefire way to take out any vamps that are somewhat low on blood, making it far more useful in the field as well, and not just its forcefeeding aspect. Vampires might need a saving grace or two in the eyes of security so-as to avoid KoS status, but I don't think the nerfs resulting from this kind of change to holy water are totally warranted.

**Comment by TullyBurnalot** [comment]
@Allfd Was just discussing this with Crazylemon, who brought up something that up until now was not actually considered: syringes. With the holy water as is with this PR, would it not turn Syringe Guns into hyper-effective vampire crippling weapons?

File: code/modules/reagents/chemistry/reagents/water.dm [hunk 1]
```diff
@@ -357,22 +357,46 @@
...
-				M.mind.vampire.nullified = max(5, M.mind.vampire.nullified + 2)
+		var/mob/living/carbon/V = M
+		if(M.mind.vampire.bloodusable)
+			M.Stuttering(1)
+			M.Jitter(30)
+			M.adjustStaminaLoss(5)
+			if(prob(20))
+				M.emote("scream")
+			M.mind.vampire.nullified = max(5, M.mind.vampire.nullified + 2)
+			M.mind.vampire.bloodusable = max(M.mind.vampire.bloodusable - 20,0)
```
Comment (Crazylemon64) [inline code/modules/reagents/chemistry/reagents/water.dm]: 20 points seems kind of extreme - a single syringe shot is 15u, 15u / (.4u/proc) = 150/4 = 37 - 37 * 20 = 800 - 60 = 740 blood lost. Meaning, a single hit from a syringe gun of holy water would be capable of nixing a vampire's abilities outright.

What if the blood loss were downed to something more reasonable? Say, to 3? At that level, they'd lose 111 blood from a syringe shot, still rather high, but not outright "game-over".

**Pull request merged**